errai-demos/errai-security-demo/pom.xml

```diff
@@ -310,6 +311,14 @@
           <extraJvmArgs>-Xmx1G -XX:CompileThreshold=7000 -Derrai.jboss.home=${errai.jboss.home} -Derrai.dev.context=${errai.dev.context}</extraJvmArgs>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <version>${maven.deploy.plugin.version}</version>
+        <configuration>
+          <skip>true</skip>
+        </configuration>
```

Comment:
@psiroky The other demos do skip the deployment, so I figured I should replicate the behavior. See https://github.com/errai/errai/blob/master/errai-jaxrs/errai-jaxrs-client/pom.xml for instance